ethereum_history_api/circuits/lib/src/verifiers/storage_test.nr

```diff
@@ -1,48 +1,64 @@
 use crate::fixtures::paris::usdc::storage_proof::storage_proofs_with_storage_root;
-use crate::verifiers::storage::{verify_storage_value};
+use crate::verifiers::storage::{verify_storage_value, verify_storage_values};
 
 #[test]
-fn test_verify_storage_proof() {
+fn test_verify_storage_value() {
     let _ = verify_storage_value(
         storage_proofs_with_storage_root.storage_root,
         storage_proofs_with_storage_root.proofs[0]
     );
 }
 
+#[test]
+fn test_verify_storage_values() {
+    let _ = verify_storage_values(
+        storage_proofs_with_storage_root.storage_root,
+        storage_proofs_with_storage_root.proofs
+    );
+}
+
 #[test(should_fail)]
```

Comment:
Unfortunately `should_fail_with="some message"` will not work here because it fails at assertion https://github.com/aragonzkresearch/noir-trie-proofs/blob/main/lib/src/lib.nr#L96 that does not have any error message

Reply:
Shall we create a PR?